index.js

```diff
@@ -160,10 +160,18 @@ export default class Camera extends Component {
 
     if (check) {
       const isAuthorized = await check();
-      this.setState({ isAuthorized });
+      if (this._isMounted) {
+        this.setState({ isAuthorized });
+      } else {
+        this.state.isAuthorized = isAuthorized;
```

Comment:
!

Reply:
What do You propose? Delete this line? Call `setState` within `setTimeout`?
How to wait `render` is finished in RN, `requestAnimationFrame`?